### PR TITLE
completed Trello board task 4

### PR DESCRIPTION
**Pull request by FlashSampson**

### CODE REVIEW

File: controller.js [hunk 1]
```diff
@@ -9,3 +9,15 @@ exports.getCategories = (req, res, next) => {
...
 };
+
+exports.getReviews = (req, res, next) => {
+  const { review_id } = req.params;
+  fetchReviews(review_id)
+    .then((data) => {
+      res.status(200).send(data);
+    })
+    .catch((err) => {
+        // console.log(err)
```
Comment (alexdevdavis) [inline controller.js:20]: can you be rid of this console.log now?

File: output.txt [hunk 2]
```diff
@@ -0,0 +1,31 @@
+You are now connected to database "nc_games_test" as user "keita_darby".
```
Comment (alexdevdavis) [inline output.txt:1]: I'd suggest gitignoring this file, as it's revealing everything about your dbs

File: psql_testing/tester.sql [hunk 3]
```diff
@@ -0,0 +1,10 @@
+\c nc_games_test;
```
Comment (alexdevdavis) [inline psql_testing/tester.sql:1]: and gitignore this too

**Review by alexdevdavis — APPROVED** [review]
Happy to approve this, with some comments to act on. Nice one!

File: controller.js (hunk 1 above)
Comment (FlashSampson) [reply controller.js:20]: Nice one - have removed now :)

File: output.txt (hunk 2 above)
Comment (FlashSampson) [reply output.txt:1]: ahh hadnt thought of that ! done

File: psql_testing/tester.sql (hunk 3 above)
Comment (FlashSampson) [reply psql_testing/tester.sql:1]: Also done :)

**Pull request merged**